pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -206,23 +209,23 @@ Our Regional Partners will be able to see de-identified demographic information
 
 ### You may share data with social networks
 
-Some of our Users choose to post their code-creations with social networks. This functionality is entirely optional. When you post content to social networks, the actual content posted is entirely at your control, and never posted automatically on your behalf. Typically this content includes only the code (app or animation or game or other) that you wrote, posted alongside any other remarks you may choose to add to it.
+Some of our Users choose to post their code-creations to social networks. This functionality is entirely optional. When you post content to social networks, the actual content posted is entirely at your control, and never posted automatically on your behalf. Typically this content includes only the code (app or animation or game or other) that you wrote, posted alongside any other remarks you may choose to add to it.
 
 ### We may share limited information when you contact us for support
 
 When you contact us with a support request, you may provide Personal Information, which is shared with a Code.org support representative in order to process your request. Code.org support representatives are either employees or independent contractors of Code.org, and will always have signed a contract requiring them to protect and not disclose confidential information including Personal Information of Users, and to use it only in the context of resolving your product support requests.
 
 ### If you are chosen to be profiled on our site
 
-Code.org promotes a weekly Student of the Week and a monthly Teacher of the Month, along with prizes. These profiles and similar testimonials are always published with the permission of the participating student or teacher and, if applicable, their parent or legal guardian, or teacher, and may include Personal Information such as the name, likeness and photo or video of the person being profiled.
+Code.org promotes Student and Teacher work on our site and social media channels. These profiles and similar testimonials are always published with the permission of the participating Student or Teacher and, if applicable, their parent or legal guardian, or teacher, and may include Personal Information such as the name, likeness and photo or video of the person being profiled.
```

Comment:
extra `or teacher` that can be removed?

Reply:
Suggest changing list to `name, likeness, photo or video`? Or is likeness tied to photo for a reason?